omod/src/main/resources/config.xml

```diff
@@ -1788,6 +1912,7 @@
 			UUID for the MONTHLY concept
 		</description>
 	</globalProperty>
+>>>>>>> 4439865bea2509904a38955cb86ea26475cf670d

```

Comment:
@KelvinWanyama this should be removed - looks like it comes from a recent merge

Reply:
I have resolved and removed this.